packages/front-end/src/app/stores/labs.ts

```diff
@@ -36,7 +36,9 @@ const useLabsStore = defineStore('labsStore', {
 
   getters: {},
 
-  persist: true,
+  persist: {
+    storage: localStorage,
```

Comment:
what's the reason for this change? it might affect the labs storage work I'm doing now

Reply:
The Nuxt persisted plugin uses cookies as the default storage mechanism, but this was failing to persist the `workflow` object - I suspect it's either a bug with the plugin or an undocumented constraint. Changing it to `localStorage` solves this problem.

This shouldn't impact your work but let me know if you run into any troubles. I'll raise another PR to make this storage change consistent across all stores.

Reply:
yeah, good idea to do that, after I posted this comment I ran into the same thing in my dev branch and used the same fix.